docs/azure_arc_jumpstart/azure_arc_servers/oci/oci_terraform_ubuntu/_index.md

```diff
@@ -0,0 +1,199 @@
+    ```
+
+    Output should look like this:
+
+    ```json
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcOCI",
+    "name": "http://AzureArcOCI",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+    > **Note: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest)**
+    
+ ## Obtain Oracle Cloud Infrastructure IDs (OCIDs)
+ 
+ In order for Terraform to create resources in OCI, you wil need to obtain the following OCID's:
+* User OCID
+* Compartment OCID
+* Tenanacy OCID
+
+ User Compartment OCID
+ * Login to the [OCI cloud console](https://www.oracle.com/cloud/sign-in.html) and select your profile. Open your user OCID, and click on show to reveal. Copy User OCID to clipboard and paste to your editor of choice.
+ 
+      ![Screenshot of OCI cloud console](./01.png)
+    
+ * After obtaining User OCID, navigate to [Compartments](https://cloud.oracle.com/identity/compartments). Click on the Tenancy name and find the OCID --then click on show. Copy the Comartment OCID to clipboard and paste it to the editor of choice..   
+ 
+    ![Screenshot of CompartmentID OCI cloud console](./02.png)
+    
+ Tenancy OCID
+* After finding the user OCID, go to [OCI Tenancy](https://cloud.oracle.com/tenancy), navigate to the OCID for Tenancy and click on show. Copy the Tenancy OCID to clipboard and paste it to the editor of choice.
+  
+   ![Screenshot of TenancyID OCI cloud console](./03.png)
+  
+ Create the API SSL Key
+ 
+ * You will need an SSL key pair to enable Terraform to connect to the OCI API under your identity. Start by generating a key
+
+    ```shell
+    $openssl genrsa -out oci_api_key.pem 2048
+    ```
+* set file access to owner only read and write
+
+  ```shell
+   $chmod go-rwx oci_api_key.pem
+  ```
+* generate the public half of the key pair
+ 
+   ```shell
+    $openssl rsa -pubout -in oci_api_key.pem -out oci_api_key_public.pem
+  ```
+
+* Copy the contents of the public key to the clipboard using gvim, pbcopy, xclip or a similar tool (you'll need to paste the value into the Console later).
+ 
+   ```shell
+     $cat oci_api_key_public.pem | gvim &
+  ```
+
+The public key needs to be added to your user account in the OCI console. Open the [account page](https://cloud.oracle.com/identity/users/) for your user, navigate to "API Keys" and select the "Add Public Key" button. Copy and paste the contents of the oci_api_key_public.pem file in the box of the "Add Public Key" dialog as shown below.
+
+  ![Screenshot of OCI cloud console](./04.png)
+
+ ## Configure Terraform
+
+Before executing the Terraform plan, you must export the environment variables which will be used by the plan. These variables are based on your Azure subscription and tenant, the Azure service principal, and the OCI OCIDs and keys you just created.
+
```

Comment:
"OCI OCIDs" ?

Reply:
What is your question?

Reply:
Suggest just say OCIDs and drop OCI in front.